hw/ip/hmac/dv/hmac_sim_cfg.hjson

```diff
@@ -68,25 +68,25 @@
     {
       name: hmac_test_sha_vectors
       uvm_test_seq: hmac_test_vectors_sha_vseq
-      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors"]
+      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors/"]
     }
 
     {
       name: hmac_test_hmac_vectors
       uvm_test_seq: hmac_test_vectors_hmac_vseq
-      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors"]
+      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors/"]
     }
 
     {
       // Append the common stress_tests.hjson entry for more run_opts.
       name: hmac_stress_all
-      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors"]
+      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors/"]
     }
 
     {
       // Append the common stress_tests.hjson entry for more run_opts.
       name: hmac_stress_all_with_rand_reset
-      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors"]
+      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors/"]
```

Comment:
HI Sri @sriyerg, I ran 2 sets of `../../../../util/dvsim.py  hmac_sim_cfg.hjson -i all_once`, no failure.
I will run a set of `all` during the weekend. Thanks for the new implementation, definitely saves tons of time.

Reply:
The weekend tests all passed as well.

Reply:
Great! thanks.